linera-chain/src/unit_tests/inbox_tests.rs

```diff
@@ -285,20 +288,22 @@ async fn test_inbox_add_then_remove_mixed() {
         .unwrap();
     view.add_event(make_event(hash, 1, 0, [2])).await.unwrap();
     // Fail to remove non-matching event (skippability).
-    assert!(matches!(
+    assert_matches!(
         view.remove_event(&make_event(hash, 0, 1, [1])).await,
         Err(InboxError::UnexpectedEvent { .. })
-    ));
+    );
     // Fail to remove non-matching event (hash).
-    assert!(matches!(
+    assert_matches!(
         view.remove_event(&make_unskippable_event(CryptoHash::new(&Dummy2), 0, 1, [1]))
             .await,
         Err(InboxError::UnexpectedEvent { .. })
-    ));
+    );
     // Fail to skip unskippable event.
-    assert!(matches!(
+    assert_matches!(
         view.remove_event(&make_event(hash, 1, 0, [2])).await,
-        Err(InboxError::UnskippableEvent {event }) if event == make_unskippable_event(hash, 0, 1, [1])));
+        Err(InboxError::UnskippableEvent {event })
```

Comment:
Optional nit: tiny formatting fix

```suggestion
        Err(InboxError::UnskippableEvent { event })
```